backend/corpora/parliament/finland-old.py

```diff
@@ -67,7 +67,7 @@ def sources(self, start, end):
     source_archive = field_defaults.source_archive()
     source_archive.extractor = CSV(field='file')
 
-    speech = field_defaults.speech()
+    speech = field_defaults.speech(language='fin')
```

Comment:
I believe 'fi' is the language string? Though 'fin' may also be valid. Have you tested with the `langcodes` module?

Reply:
It's `fi` in ISO 631-1 and `fin` in 639-3. I think `langcodes` accepts either, but we don't apply standardisation to the names of analysers in elasticsearch, so this could cause issues.

In any case, I didn't know there was also a lot of Swedish, so you're right, the language should not be set like this.

Reply:
You would need to override the field default of `es_mappings` though, since that has stopword removal set to `True`, which would fail without the language information.